### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Add items
https://www.divine-pride.net/database/item/1320
https://www.divine-pride.net/database/item/1325
https://www.divine-pride.net/database/item/1444
https://www.divine-pride.net/database/item/1842
https://www.divine-pride.net/database/item/1843
https://www.divine-pride.net/database/item/1945
https://www.divine-pride.net/database/item/1946
https://www.divine-pride.net/database/item/2035
https://www.divine-pride.net/database/item/2036
https://www.divine-pride.net/database/item/13130
https://www.divine-pride.net/database/item/13140
https://www.divine-pride.net/database/item/13141
https://www.divine-pride.net/database/item/13333
https://www.divine-pride.net/database/item/13334
https://www.divine-pride.net/database/item/13466
https://www.divine-pride.net/database/item/13467
https://www.divine-pride.net/database/item/15187
https://www.divine-pride.net/database/item/15188
https://www.divine-pride.net/database/item/16049
https://www.divine-pride.net/database/item/16058
https://www.divine-pride.net/database/item/16059
https://www.divine-pride.net/database/item/18133
https://www.divine-pride.net/database/item/18146
https://www.divine-pride.net/database/item/18147
https://www.divine-pride.net/database/item/20827
https://www.divine-pride.net/database/item/20828
https://www.divine-pride.net/database/item/21026
https://www.divine-pride.net/database/item/21027
https://www.divine-pride.net/database/item/22127
https://www.divine-pride.net/database/item/22128
https://www.divine-pride.net/database/item/26004
https://www.divine-pride.net/database/item/26005
https://www.divine-pride.net/database/item/26105
https://www.divine-pride.net/database/item/26106
https://www.divine-pride.net/database/item/26143
https://www.divine-pride.net/database/item/26144
https://www.divine-pride.net/database/item/26201
https://www.divine-pride.net/database/item/26202
https://www.divine-pride.net/database/item/27031
https://www.divine-pride.net/database/item/27032
https://www.divine-pride.net/database/item/28018
https://www.divine-pride.net/database/item/28019
https://www.divine-pride.net/database/item/28117
https://www.divine-pride.net/database/item/28118
https://www.divine-pride.net/database/item/28227
https://www.divine-pride.net/database/item/28228
https://www.divine-pride.net/database/item/28229
https://www.divine-pride.net/database/item/28230
https://www.divine-pride.net/database/item/28231
https://www.divine-pride.net/database/item/28232
https://www.divine-pride.net/database/item/28233
https://www.divine-pride.net/database/item/28234
https://www.divine-pride.net/database/item/28609
https://www.divine-pride.net/database/item/28610
https://www.divine-pride.net/database/item/28708
https://www.divine-pride.net/database/item/28722
https://www.divine-pride.net/database/item/28723
https://www.divine-pride.net/database/item/32001
https://www.divine-pride.net/database/item/32002

Data correction for item combo IDs 2053:19451
https://www.divine-pride.net/database/item/2053
https://www.divine-pride.net/database/item/19451

Data correction for item IDs 500007
https://www.divine-pride.net/database/item/500007

### CODE REVIEW

**Comment by mazvi** [comment]
Please include this https://github.com/rathena/rathena/issues/6091

**Comment by mazvi** [comment]
Need update again about item ID 22107 because on item_db_equip.yml inconsistent with other boots 

like another boots AGI DEX INT LUK VIT

https://www.divine-pride.net/database/item/22107/modified-str-boots

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+    Jobs:
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(7*readparam(bInt)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111665]: Precedence, the result is different
```suggestion
      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(readparam(bInt)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bBaseAtk,2*getskilllv("GS_SNAKEEYE");
+      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(7*readparam(bInt)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111695]:
```suggestion
      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(readparam(bInt)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+    Jobs:
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSkillAtk,"RL_R_TRIP",(7*readparam(bAgi)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111724]:
```suggestion
      bonus2 bSkillAtk,"RL_R_TRIP",(readparam(bAgi)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bBaseAtk,2*getskilllv("GS_SNAKEEYE");
+      bonus2 bSkillAtk,"RL_R_TRIP",(7*readparam(bAgi)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111754]:
```suggestion
      bonus2 bSkillAtk,"RL_R_TRIP",(readparam(bAgi)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+    Jobs:
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSkillAtk,"RL_D_TAIL",(7*readparam(bLuk)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111783]:
```suggestion
      bonus2 bSkillAtk,"RL_D_TAIL",(readparam(bLuk)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bBaseAtk,2*getskilllv("GS_SNAKEEYE");
+      bonus2 bSkillAtk,"RL_D_TAIL",(7*readparam(bLuk)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111813]:
```suggestion
      bonus2 bSkillAtk,"RL_D_TAIL",(readparam(bLuk)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bSplashRange,1;
+      bonus2 bSkillAtk,"RL_SLUGSHOT",(7*readparam(bStr)/10);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111843]:
```suggestion
      bonus2 bSkillAtk,"RL_SLUGSHOT",(readparam(bStr)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 8]
```diff
@@ -110468,6 +111644,244 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:111873]:
```suggestion
      bonus2 bSkillAtk,"RL_SLUGSHOT",(readparam(bStr)/10)*7;
```

File: db/re/item_db_equip.yml [hunk 9]
```diff
@@ -114447,6 +115964,98 @@ Body:
...
+      Merchant: true
+      Ninja: true
+      Novice: true
+      Rogue: true
+      Sage: true
+      SoulLinker: true
+      SuperNovice: true
+      Swordman: true
+      Thief: true
+      Wizard: true
```
Comment (Atemo) [inline db/re/item_db_equip.yml:115997]:
```suggestion
      Wizard: true
    Classes:
      All_Third: true
```

File: db/re/item_db_equip.yml [hunk 10]
```diff
@@ -114447,6 +115964,98 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:116043]:
```suggestion
      Wizard: true
    Classes:
      All_Third: true
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111665]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111695]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111724]: Thanks

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111754]: Thanks

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111783]: Thanks

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111813]: Thanks

File: db/re/item_db_equip.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111843]: Thanks

File: db/re/item_db_equip.yml (hunk 8 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:111873]: Thanks

File: db/re/item_db_equip.yml (hunk 9 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:115997]: Thanks

File: db/re/item_db_equip.yml (hunk 10 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**